tsconfig.json

```diff
@@ -0,0 +1,6 @@
+{
+  "compilerOptions": {
+    "experimentalDecorators": true,
+    "allowJs": true
+  }
+}
```

Comment:
Dummy file to avoid Vetur error during development, suggested by https://github.com/vuejs/vetur/issues/815#issuecomment-408026341